runtime_builders/publish_builders.py

```diff
@@ -63,9 +63,8 @@ def _publish_latest(builder_dir):
         project_name = config['project']
 
         parts = os.path.splitext(latest)
-        project_prefix = os.path.join(builder_util.RUNTIME_BUCKET_PREFIX,
-                                      project_name, '-')
-        latest_file = parts[0][len(project_prefix):]
+        prefix = builder_util.RUNTIME_BUCKET_PREFIX + project_name + '-'
```

Comment:
This way will work as long as builder_util.RUNTIME_BUCKET_PREFIX is hardcoded and doesn't change. os.path.join should still work in that case, if you instead concat project_name and the hyphen. not sure that's buying us a whole lot, so I think I'm alright with a straight string concat here.